### PR TITLE
Remove pro v1

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**
- [ ] I added an entry to [`CHANGELOG.md`](CHANGELOG.md) if knowledge of this change could be valuable to users.

---

Here is a brief summary of what I did:

1. remove pro feature
2. change CI

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[`pro` entfernen](https://trello.com/c/6wGy0O0E/1723-pro-entfernen)

File: services/src/server.rs [hunk 1]
```diff
@@ -108,7 +67,7 @@ where
         api = serve_openapi_json(
             api,
             &mut api_urls,
-            "Geo Engine",
+            "Geo Engine Pro",
```
Comment (michaelmattig) [inline services/src/server.rs:70]: Geo Engine Pro doesn't exist anymore.

File: .github/workflows/containers.yml [hunk 2]
```diff
@@ -60,6 +63,7 @@ jobs:
         uses: actions/checkout@v3
         with:
           repository: geo-engine/geoengine-container
+          ref: ${{ env.CONTAINER_REPOSITORY_BRANCH }}
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply .github/workflows/containers.yml:66]: it's not this repo, but the container repo

File: cli/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+[package]
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply cli/Cargo.toml]: @ChristianBeilschmidt
[remove crate cli](https://github.com/geo-engine/geoengine/pull/999/commits/92e6e4338e30c2a2329f7734e956ab1aea4c105e)

File: services/src/server.rs (hunk 1 above)
Comment (ChristianBeilschmidt) [reply services/src/server.rs]: [remove pro notions](https://github.com/geo-engine/geoengine/pull/999/commits/6804675c8db2782ddc72a08db1e05694e889f662)

File: Settings-default.toml [hunk 4]
```diff
@@ -144,11 +144,6 @@ allowed_drivers = [
 # Whether to allow requests to `/anonymous` that return a valid session.
 anonymous_access = true
 
-# Specifies a fixed session token. Otherwise, a random token is generated.
```
[the remark this answers is not in the document]
Comment (ChristianBeilschmidt) [reply Settings-default.toml:147]: [remove session.fixed_session_token from test](https://github.com/geo-engine/geoengine/pull/999/commits/fd233d3b3685da477b7892a80bf5551af9c2c96f)

**Review by michaelmattig — APPROVED** [review]

**Pull request merged**